app/assets/stylesheets/base/color_and_text_styles.scss

```diff
@@ -9,6 +9,10 @@ title {
   letter-spacing: 0.2px;
   color: $text-title;
 }
+label {
+  @extend .body-medium;
```

Comment:
typically `@extend` is discouraged due to how it can get really messy with potentially nested classes. You usually want to make a mixin instead